CogNative/backend/translation/translation.py

```diff
@@ -0,0 +1,15 @@
+from googletrans import Translator

```

Comment:
Add the correct version of googletrans in requirements.txt

Please add a _version_! When I installed googletrans with pip the default version didn't run this code properly.